src/um/audiosessiontypes.rs

```diff
@@ -27,9 +27,12 @@ pub const AUDCLNT_STREAMFLAGS_LOOPBACK: DWORD = 0x00020000;
 pub const AUDCLNT_STREAMFLAGS_EVENTCALLBACK: DWORD = 0x00040000;
 pub const AUDCLNT_STREAMFLAGS_NOPERSIST: DWORD = 0x00080000;
 pub const AUDCLNT_STREAMFLAGS_RATEADJUST: DWORD = 0x00100000;
+pub const AUDCLNT_STREAMFLAGS_PREVENT_LOOPBACK_CAPTURE: DWORD = 0x01000000;
```

Comment:
I can't find this constant in the Windows 10 SDK.

Reply:
removed.